app/models.py

```diff
@@ -843,6 +848,16 @@ def validates_status(self, key, status):
             self.published_at = datetime.utcnow()
         return status
 
+    def add_clarification_question(self, question, answer):
+        clarification_question = BriefClarificationQuestion(
+            brief=self,
+            question=question,
+            answer=answer)
+
+        Session.object_session(self).add(clarification_question)
```

Comment:
It gets the SQLAlchemy session associated with the `Brief` object (ie. the session that was used to retrieve it) and then adds the `BriefClarificationQuestion` object to it.